apps/docs/babel.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  presets: [require.resolve('@docusaurus/core/lib/babel/preset')],
```

Comment:
was the deep import preset auto generated ?

Reply:
Yup